### PR TITLE
i18n(es): update `core-concepts/routing` page

### PR DESCRIPTION
**Pull request by antonyfaris**
#### What kind of changes does this PR include?

- Translated content
- New or updated content

#### Description

-  Update with [these changes](https://github.com/withastro/docs/commit/bdd67f186fa39e0159840ca995722b409b7b3af5)

### CODE REVIEW

File: src/pages/es/core-concepts/routing.md [hunk 1]
```diff
@@ -337,3 +337,26 @@ export async function getStaticPaths({paginate}) {
 const { page } = Astro.props;
 const params = Astro.params;
 ```
+
+
+## Excluyendo páginas
+
+Puedes excluir páginas, o incluso directorios completos de ser generadas añadiendo el prefijo (`_`).
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/routing.md:344]:
```suggestion
Puedes excluir páginas, o incluso directorios completos de ser generados añadiendo el prefijo (`_`).
```

File: src/pages/es/core-concepts/routing.md [hunk 2]
```diff
@@ -337,3 +337,26 @@ export async function getStaticPaths({paginate}) {
...
+
+Esto permite crear páginas privadas, y también coubicar *tests*, utilidades, y componentes con su páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```
Comment (agustinmulet) [inline src/pages/es/core-concepts/routing.md:346]:
```suggestion
Esto permite crear páginas privadas, y también co ubicar *tests*, utilidades y componentes con su páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```

**Review by agustinmulet — COMMENTED** [review]
Left a couple of suggestions, nice work!

**Review by agustinmulet — APPROVED** [review]
LGTM! ✅

File: src/pages/es/core-concepts/routing.md [hunk 3]
```diff
@@ -337,3 +337,26 @@ export async function getStaticPaths({paginate}) {
 const { page } = Astro.props;
 const params = Astro.params;
 ```
+
+
+## Excluyendo páginas
+
+Puedes excluir páginas, o incluso directorios completos de ser generados añadiendo el prefijo (`_`).
+
+Esto permite crear páginas privadas, y también co ubicar *tests*, utilidades y componentes con su páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```
Comment (fredoist) [inline src/pages/es/core-concepts/routing.md:346]: I was not able to find a definition for "co ubicar" in Spanish, I think just "colocar" fits better in this paragraph.

```suggestion
Esto permite crear páginas privadas, y también colocar *tests*, utilidades y componentes con sus páginas relacionadas, evitando generar archivos `.html` en el directorio `dist/`.
```

**Review by fredoist — CHANGES_REQUESTED** [review]
Reviewed

File: src/pages/es/core-concepts/routing.md (hunk 3 above)
Comment (antonyfaris) [reply src/pages/es/core-concepts/routing.md:346]: @agustinmulet

Comment (kevinzunigacuellar) [reply src/pages/es/core-concepts/routing.md:346]:
```suggestion
Esto te permite crear páginas privadas, y también incluir otros tipos de archivos como *tests*, utilidades y componentes junto con las páginas a donde pertenecen, evitando generar archivos `.html` en el directorio `dist/`.
```

**Pull request merged**